### ISSUE TITLE
Python import error when launching stream

### ISSUE THREAD

**Issue by noahleigh** [streamlink/streamlink-twitch-gui#373]
### Description

Just installed Streamlink 0.2.0 and Streamlink Twitch GUI 1.0.1 and when I try to launch a stream, I get an error message:

> **Error: Couldn't launch the stream**
> Unexpected version check output

I can launch a stream with the Streamlink commandline interface just fine (`streamlink twitch.tv/ESL_overwatch source`), so I'm making an issue here instead of the Streamlink repo.

### Expected / Actual behavior

Stream should launch

### Reproduction steps

1. Try to launch a stream

### Environment details (operating system, etc.)

- Windows 10 Pro x64 Build 14965
- Python 2.7.11 64-bit
- Streamlink 0.2.0
- Streamlink Twitch GUI 1.0.1 64-bit (installed)
- Default Streamlink settings in the GUI

### Comments, logs, screenshots, etc.

Traceback:
```
Traceback (most recent call last):
File "C:\Program Files (x86)\Streamlink\bin\streamlink-script.py", line 11, in <module>
from streamlink_cli.main import main
File "C:\Program Files (x86)\Streamlink\pkgs\streamlink_cli\main.py", line 17, in <module>
from streamlink.stream import StreamProcess
File "C:\Program Files (x86)\Streamlink\pkgs\streamlink\stream\__init__.py", line 6, in <module>
from .hls import HLSStream
File "C:\Program Files (x86)\Streamlink\pkgs\streamlink\stream\hls.py", line 4, in <module>
from Crypto.Cipher import AES
File "C:\Program Files (x86)\Streamlink\pkgs\Crypto\Cipher\__init__.py", line 78, in <module>
from Crypto.Cipher._mode_ecb import _create_ecb_cipher
File "C:\Program Files (x86)\Streamlink\pkgs\Crypto\Cipher\_mode_ecb.py", line 46, in <module>
"""
File "C:\Program Files (x86)\Streamlink\pkgs\Crypto\Util\_raw_api.py", line 168, in load_pycryptodome_raw_lib
raise OSError("Cannot load native module '%s'" % name)
OSError: Cannot load native module 'Crypto.Cipher._raw_ecb'
```

**Comment by bastimeyer** [comment]
> Python 2.7.11 64-bit

If you don't set an explicit python path in the settings, the GUI will look for the first path registered in your system's `PATH` env var. Since you've installed python 2.7 in addition to Streamlink (via the installer which uses python 3.5), it will try to use python 2.7, together with streamlink and its dependencies built for python 3.5.
You either need to install streamlink via pip (expecting that the `Scripts` python folder is defined in your `PATH` env var before the streamlink install dir), or you need to set a custom path to the included python 3.5 executable in the setting menu.

> streamlink twitch.tv/ESL_overwatch source

This will execute `streamlink.exe` provided by the installer (unless streamlink has been installed via python-pip)

Here's a description of what the GUI is doing (which is also described in the settings, btw):
https://github.com/streamlink/streamlink-twitch-gui/wiki/Streamlink-configuration#selecting-a-streaming-provider

**Comment by noahleigh** [comment]
Yep, picking the `pythonw.exe` packaged with Streamlink did the trick, thanks!
